.github/workflows/ci.yaml

```diff
@@ -2,12 +2,12 @@ name: "CI"
 on:
   pull_request:
```

Comment:
also I updated this to be more strict ( it was "Require approval for first-time contributors" before): 

<img width="681" height="462" alt="Image" src="https://github.com/user-attachments/assets/397d7903-dee9-4fb4-b921-fd1fff2ef431" />

and then we can use: https://docs.github.com/en/actions/how-tos/manage-workflow-runs/approve-runs-from-forks

Reply:
That's fine, I figured a trust on first use model (like the one we use for `ssh`) was okay, but I don't think we can be too careful. It does mean we can be quicker to let someone merge in a one line typo change since it then doesn't get them into the "run without approval" list